digitalidentity/policy_builder.go

```diff
@@ -0,0 +1,250 @@
+package digitalidentity
```

Comment:
It will have to be reworked in next stages but for now it will stay like this.

Reply:
@mehmet-yoti the point i was getting at is that the ticket https://lampkicking.atlassian.net/browse/SDK-2240 and the PR description is literally empty, i'd expect a note/ mention of what's expected to be done and a note that the `dynamic` package is being duplicated for now because of future plans to deprecate it.